### ISSUE TITLE
Allow option to omit __typename from generated interfaces

### ISSUE THREAD

**Issue by maxcan** [avantcredit/gql2ts#42]
So I'm using gql2ts to generate interfaces to statically typecheck my resolver functions.  One issue is that those functions should not populate the `__typename` field in the objects they return but then the returned objects fail to typecheck.  

Would there be interest in adding an option to omit the `__typename` fields or possibly include them in a subtype?  If so, I'll write and send a PR.

so, instead of: 
```typescript
interface IUser {
    __typename: string;
    id: string;
    email: string;
}
```
we'd have something like:
```typescript
interface IUser_NoTypename {
    id: string;
    email: string;
}

interface IUser extends IUser_NoTypename {
    __typename: string;
}
```

**Comment by brettjurgens** [comment]
sorry for the late reply, I'm not sure the best way to do this. But if you want to open a PR, I'll take a look 😄

**Comment by brettjurgens** [comment]
fixed in #72
